src/lib.rs

```diff
@@ -29,7 +29,7 @@ impl OwnedLabel {
 }
 
 lazy_static::lazy_static! {
-    static ref GLOBAL: Arc<Global> = Arc::new(Global::new("wgpu", wgc::hub::IdentityManagerFactory));
+    static ref GLOBAL: Arc<Global> = Arc::new(Global::new("wgpu", wgc::hub::IdentityManagerFactory, wgt::BackendBit::PRIMARY));
```

Comment:
Whats the best way to handle setting the backend bit?

Reply:
Good question! I think this should be come the `Instance` soon. We haven't discussed yet how the backends will be selected at this stage....